src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -911,7 +911,7 @@
   - name: Create an Openshift cluster using a custom vnet
     text: az openshift create -g MyResourceGroup -n MyManagedCluster --vnet-peer "/subscriptions/0000000-0000-0000-0000-000000000000/resourceGroups/openshift-vnet/providers/Microsoft.Network/virtualNetworks/test"
   - name: Create an Openshift cluster with Log Analytics monitoring enabled
-    text: az openshift create -g MyResourceGroup -n MyManagedCluster --workspace-resource-id {WORKSPACE_RESOURCE_ID}
+    text: az openshift create -g MyResourceGroup -n MyManagedCluster --workspace-id "/subscriptions/0000000-0000-0000-0000-000000000000/resourceGroups/<resourcegroup>/providers/Microsoft.OperationalInsights/workspaces/<workspace-resource-id>"
```

Comment:
nit: would it be better to use `MyResourceGroup` or `MyRG` style instead of `<resourcegroup>? Other help text strings that include ARM ID don't use `<>` (`git grep "00000000-0000-0000-0000-000000000000" | grep -v tests`)